packages/web/src/experience/Markdown.tsx

```diff
@@ -0,0 +1,94 @@
+import MarkdownJSX from 'markdown-to-jsx'
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import PlanningDocs from 'src/experience/eventkit/PlanningDocs'
+import {
+  DesignForAll,
+  EmbodyHumility,
+  InnovatingOnMoney,
+  StrivingForBeauty,
+} from 'src/experience/eventkit/Tenent'
+import { H1, H2, H3, H4, Li, Ul } from 'src/fonts/Fonts'
+import Button, { BTN } from 'src/shared/Button.3'
+import InlineAnchor from 'src/shared/InlineAnchor'
+import { fonts, standardStyles } from 'src/styles'
+import { trackOpen, Types } from './eventkit/tracking'
+
+function P({ children }) {
+  return <Text style={[fonts.p, standardStyles.halfElement, styles.block]}>{children}</Text>
+}
+
+function PrimeButton({
+  children,
+  href,
+  external,
+}: {
+  children: string
+  href: string
+  external?: boolean
+}) {
+  const track = React.useCallback(() => trackOpen({ name: children, type: Types.Action }), [
+    children,
+  ])
+
+  return (
+    <Button
+      style={standardStyles.halfElement}
+      kind={BTN.PRIMARY}
+      text={children}
+      href={href}
+      onPress={track}
```

Comment:
Is track the only thing that happens on press?

Reply:
Of is this an anchor (I see there's an href above)?

Reply:
yes its an anchor.